temba/msgs/tests.py

```diff
@@ -2103,3 +2107,120 @@ def test_render(self):
         # exception if tag not used correctly
         self.assertRaises(ValueError, self.render_template, '{% load sms %}{% render with bob %}{% endrender %}')
         self.assertRaises(ValueError, self.render_template, '{% load sms %}{% render as %}{% endrender %}')
+
+
+class CeleryTaskTest(TransactionTestCase):
```

Comment:
I think you could avoid a lot of the boilerplate of setting up an org etc, if you just subclass `TembaTest`